ivy/array/utilities.py

```diff
@@ -1,18 +1,17 @@
-# global
+
```

Comment:
There should be no changes to this file, including changing the file name. Please revert any changes you have made to this file.

Reply:
done

Reply:
There are still changes in the file. You should not be changing anything in ivy/array/utility when doing the Frontend Task